app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="margin_small">8dp</dimen>
```

Comment:
đặt là dp_8 thay vì margin_small. Sau nếu ô dùng kích thước này cho padding hay cái khác thì sẽ không ổn

Reply:
Thank you